CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Mapbox welcomes participation and contributions from everyone.
 # main
 ## Features ✨ and improvements 🏁
 * Deprecated gesture settings `increaseRotateThresholdWhenPinchingToZoom` property. ([1632](https://github.com/mapbox/mapbox-maps-android/pull/1632))
+* Add support to set location puck opacity. ([1659](https://github.com/mapbox/mapbox-maps-android/pull/1659))
```

Comment:
```suggestion
* Add location puck opacity support. ([1659](https://github.com/mapbox/mapbox-maps-android/pull/1659))
```

Reply:
3D puck had opacity support before already so guess worth pointing it out that it's for 2D puck only

Reply:
Actually it's for both. AFAIK it was not working for the 3D one. See changes here https://github.com/mapbox/mapbox-maps-android/pull/1659/files#diff-3e3c552e13138daef0e5a55df747ed7d4364d9da9bf67b2554c9c0190d2782a7R32